openfisca_us/tests/policy/baseline/irs/credits/child_tax_credit/maximum.yaml

```diff
@@ -0,0 +1,21 @@
+- name: Pre-TCJA amounts
+  period: 2017
+  input:
+    age: [4, 16, 17]
+    is_tax_unit_dependent: [true, true, true]
+  output:
+    ctc_individual_maximum: [1_000, 1_000, 0]
+- name: Post-TCJA, pre-ARPA amounts
```

Comment:
nits but i think we're naming the unit tests as the name of the variable, so the folder structure matches the variable folder structure

could you also add an empty line between tests?